### PR TITLE
GH avatar sizes

### PR DESCRIPTION
**Pull request by i386**
# Description

Only serve up github avatars at 50x50px to reduce size. As @cliffmeyers discovered some avatars can be unreasonably big.

PTAL @vivek @cliffmeyers 

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/AbstractGithubOrganization.java [hunk 1]
```diff
@@ -5,13 +5,24 @@
...
+import java.util.logging.Logger;
+
 /**
  * @author Vivek Pandey
  */
 public abstract class AbstractGithubOrganization extends ScmOrganization {
 
+    private static final Logger LOGGER = Logger.getLogger(AbstractGithubOrganization.class.getName());
+
+    static final int AVATAR_SIZE = 50;
```
Comment (vivek) [inline blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/AbstractGithubOrganization.java]: Don't see anyone else using it, want to make it private?

**Comment by vivek** [comment]
@i386 There is one comment about package private vs private. Otherwise LGTM 🐝  pending CI and ATH run.

**Comment by i386** [comment]
@vivek ack

**Comment by i386** [comment]
This code is not exercised at all by the ATH. Merging.

**Pull request merged**